jscomp/core/js_of_lam_variant.ml

```diff
@@ -84,3 +84,13 @@ let eval_as_int (arg : J.expression) (dispatches : (int * int) list ) : E.t  =
                body = [S.return (E.int (Int32.of_int  r))],
                       false (* FIXME: if true, still print break*)
               }) dispatches))]
+
+let eval_as_unwrap (arg : J.expression) : E.t =
+  if arg == E.undefined then
+    E.undefined
+  else
```

Comment:
how would you generate code for such case?
```ocaml
external f : ?x:[`a of int] -> int = "" [@@bs.val]
let u = f
```
I suppose it would generate code like this
```js
function u (prim){
   return f(prim?prim[0] [1] : undefined)
}
```
if so can you add it